lib/src/subscription.dart

```diff
@@ -76,7 +74,7 @@ class SubscriptionImpl implements Subscription {
   @override
   void subscribe() {
     _state = _SubscriptionState.subscribed;
-    if (!_client.connected) {
+    if (!_client.connected!) {
```

Comment:
Since the `connected` property can't be null, '!' is not needed here.

Reply:
Fixed